react-motion/build.boot

```diff
@@ -48,8 +48,6 @@
                  "cljsjs/react-motion/development/react-motion.inc.js"})
     (minify :in "cljsjs/react-motion/development/react-motion.inc.js"
             :out "cljsjs/react-motion/production/react-motion.min.inc.js")
-    (sift :include #{#"^cljsjs"})
-    (deps-cljs :name "cljsjs.react-motion"
```

Comment:
Please use `deps-cljs` task to ensure correct file paths and `validate` task to ensure the JS syntax is valid: https://github.com/cljsjs/packages/blob/master/create-react-class/build.boot#L25-L30